src/models/draft.js

```diff
@@ -17,6 +17,8 @@ export default class Draft extends Message {
     if (this.draft) {
       json.object = 'draft';
     }
+
+    json['tracking'] = this.tracking;
```

Comment:
Thinking about where it makes sense to add this --

`tracking` isn't really an attribute on a Draft object (or a Message object), so defining it inside of `toJSON()` would be a little strange. The `toJSON` method is used in a lot of places like `saveRequestBody()`, which means that changing `toJSON` would have unintended consequences, including sending that `tracking` object to the POST /drafts endpoint when creating a draft, which we don't want to do.

`saveRequestBody()` makes a little more sense, because it's specifically used to generate the body of a request, but since it's used to generate a `save` request (creating the draft), it would be a little weird there, too.

I'm thinking it makes the most sense to have `tracking` (or maybe even `track_opens`, `track_link_clicks`, etc.) be an argument to the `.send()` method, because that method only talks to the /send endpoint, which is the only endpoint that needs/wants tracking data. Does that sound good to you?

Reply:
I have the exact same thoughts as @Gerdie. Our `tracking` attribute is only available in our API through our sending functionality: https://docs.nylas.com/reference#message-tracking-overview. 
So I agree with @Gerdie on having `tracking` as an argument to the `.send()` method, but I think we can keep it as `tracking` and just have `tracking` be an object with the different types of tracking functionality we have, similar to how it's shown in our docs that I linked above.

Reply:
makes sense, thank you for the feedback !